src/api/libcellml/issue.h

```diff
@@ -103,15 +103,15 @@ class LIBCELLML_EXPORT Issue
         COMPONENT,
         CONNECTION,
         ENCAPSULATION,
+        GENERATOR,
```

Comment:
Agreed, but I was merely putting it in the right place so that the listing is alphabetically ordered, as it should have been in the first place. `GENERATOR` is properly removed in PR #631 (see [here](https://github.com/cellml/libcellml/pull/631/files#diff-d7a946aa2f3f6a627b853150866642f8)).